gdk4/src/lib.rs

```diff
@@ -65,3 +68,39 @@ pub use rectangle::Rectangle;
 pub use rgba::{RgbaParseError, RGBA};
 pub use surface::SurfaceExtManual;
 pub use time_coord::TimeCoord;
+
+// This is the priority that the idle handler processing surface updates is given in the glib::MainLoop.
+#[doc(alias = "GDK_PRIORITY_REDRAW")]
+pub const PRIORITY_REDRAW: u32 = ffi::GDK_PRIORITY_REDRAW as u32;
+
+// A mask covering all entries in ModifierType.
+#[doc(alias = "GDK_MODIFIER_MASK")]
+pub const MODIFIER_MASK: u32 = ffi::GDK_MODIFIER_MASK as u32;
```

Comment:
Isn't this the same as `gdk::ModifierType::all()`?

Reply:
Ah Indeed. Does it make sense to re-export then as I think that `gdk::ModifierType::all()` is less discoverablee than `MODIFIER_MASK`

Reply:
Well, at least export it with the correct type :)